duckduckgo/Android
# Update project dependencies

Task/Issue URL: https://app.asana.com/0/414730916066338/1199916162915091/f
Tech Design URL: 
CC: 

**Description**:
Updates project dependencies for:
1. Kotlin
2. Android Gradle Plugin
3. Anvil
4. Coroutines

**Steps to test this PR**:
1. Perform a clean build and assemble from command line; verify builds fine
1. Perform a clean build and assemble from Android Studio; verify app builds and installs
2. Perform a general smoke test of the app trying out various sites and features to double check there's no side effects


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
